### ISSUE TITLE
Ordenación establecimientos de pago

### ISSUE THREAD

**Issue by vicius23** [compropago/plugin-woocommerce#71]
Gracias a todo el equipo por el soporte de esta semana y por las excelentes updates.

Echo en falta una función, que sería la posibilidad de ordenar los establecimientos de pago para que al cliente le salga en primeros lugares otros de los que son por defecto, lo indico por si se considera oportuno desarrollarla para futuras versiones.

Un saludo.

**Comment by dabodude** [comment]
+1 por favor. Todos quieren Oxxo y lo primero que sale es 7Eleven.

**Comment by liberaiphoneimei** [comment]
Exacto, ese es principalmente el motivo. A ver si desde ComproPago nos echan una mano y nos dejan ordenarlos ;-)

**Comment by liberaiphoneimei** [comment]
Con Oxxo de primero puede servir, sí ;-) ¡Gracias!

**Comment by dabodude** [comment]
Por favor. Gracias!
